src/trace/context.ts

```diff
@@ -238,6 +238,89 @@ export function readTraceFromSQSEvent(event: SQSEvent): TraceContext | undefined
   return;
 }
 
+export function readTraceFromSNSSQSEvent(event: SQSEvent): TraceContext | undefined {
```

Comment:
Out of curiosity, in dd-lambda-js do we have multiple `readTraceFrom...Event` functions that are specifically tailored for each aws managed service? 
If so then why do we have a readTraceFromSQSEvent [function](https://github.com/DataDog/datadog-lambda-js/blob/8b5af7e53f6de008139f44c6801e0724a08abab9/src/trace/context.ts#L198) and readTraceFromSNSSQSEvent (line 241)? 

Is an SQS event and SNSSQS event different? Therefore needing two separate functions?

Reply:
Yeah they are a bit different, basically the SNSSQS event is not batched, but a typical SNS event has an array of `Records` with each SNS message inside it. I think we could probably refactor this though, going to take a closer look